src/nvme/util.h

```diff
@@ -169,10 +169,10 @@ int nvme_get_feature_length(int fid, __u32 cdw11, __u32 *len);
  * @NVME_DATA_TFR_BIDIRECTIONAL:	Bidirectional
  */
 enum nvme_data_tfr {
-	NVME_DATA_TFR_NO_DATA_TFR,
-	NVME_DATA_TFR_HOST_TO_CTRL,
-	NVME_DATA_TFR_CTRL_TO_HOST,
-	NVME_DATA_TFR_BIDIRECTIONAL
+	NVME_DATA_TFR_NO_DATA_TFR   = 0x0,
+	NVME_DATA_TFR_HOST_TO_CTRL  = 0x1,
+	NVME_DATA_TFR_CTRL_TO_HOST  = 0x2,
+	NVME_DATA_TFR_BIDIRECTIONAL = 0x3
```

Comment:
I also asked to move this to the right header. Don't bother. I'll do this part myself now to avoid that release from later today has this part wrong.

As not all points have been addressed by this update I wont pull it.